cadquery/cq.py

```diff
@@ -3675,6 +3648,26 @@ def toPending(self) -> "Workplane":
 
         return self
 
+    def offset2D(
+        self, d: float, kind: Literal["arc", "intersection", "tangent"] = "arc"
+    ) -> "Workplane":
+        """
+        Creates a 2D offset wire.
+        
+        :param float d: thickness. Negative thickness denotes offset to inside.
```

Comment:
So this implies that a 0 offset can be used, even though it's not practical to do so. Does the kernel handle that properly?

Reply:
0 offset just works